engineapi/engine.go

```diff
@@ -268,6 +259,59 @@ func (e *EngineBinary) ReplicaRebuildVerify(engine *longhorn.Engine, url string)
 	return nil
 }
 
+func (e *EngineBinary) GetLogger() logrus.FieldLogger {
+	return &logrus.Entry{}
+}
+
+func (e *EngineBinary) IsGRPC() bool {
+	return false
+}
+
+func (e *EngineBinary) GetConnectionState() connectivity.State {
+	return connectivity.Idle
+}
+
+func (e *EngineBinary) Start(int, *longhorn.InstanceManager, logrus.FieldLogger, *datastore.DataStore, chan struct{}) error {
+	return errors.Errorf(ErrNotImplement)
```

Comment:
Can we just panic with "not implemented" message? because this is actually on purpose panic and callers should not call them. 

The same as others.

Reply:
Updated.